### ISSUE TITLE
Fix webapp pylint errors

### ISSUE THREAD

**Issue by williexu** [Azure/azure-cli-extensions#166]
### Webapp 

Run ./scripts/ci/test_static.sh on the [pylint](https://github.com/Azure/azure-cli-extensions/tree/pylint) branch
Pylint violations: https://travis-ci.org/Azure/azure-cli-extensions/jobs/375143935

-----

**Comment by williexu** [comment]
@panchagnula when can this be done? All extensions have been cleaned of pylint errors aside from webapp.
Changes shouldn't take long.

**Comment by panchagnula** [comment]
@williexu , can you give me instructions here to identify the fixes? i did a few updates on Sunday, however, since the Travis CI didn't fail i wasn't sure what all needed to be updates. Thanks!

**Comment by williexu** [comment]
@panchagnula yep, I've synced the pylint branch with master: https://github.com/Azure/azure-cli-extensions/commits/pylint

The CI for that branch is currently failing atm: https://travis-ci.org/Azure/azure-cli-extensions/builds/377068460?utm_source=github_status&utm_medium=notification (you can reach this page by clicking the red X next to the latest commit)
You can also identify the fixes needed by running `./scripts/ci/test_static.sh` on your machine after checking out the [pylint](https://github.com/Azure/azure-cli-extensions/commits/pylint) branch.

Please make a PR against that branch after addressing the violations.

Hope this helps, also feel free to DM me with any other questions.

**Comment by williexu** [comment]
Addressed with https://github.com/Azure/azure-cli-extensions/pull/175
